File: gljive.c
#include<stdio.h>
int main()
{
	int s,i=0,ans = 0,temp;
	while (i < 10)
	{
		i++;
		scanf("%d",&s);
		temp = ans;
		ans += s;
		if (100 > ans)
			continue;
		else if (ans == 100)
			break;
		else if ((ans - 100) == (100 - temp))
			break;
		else if ((ans - 100) > (100 - temp))
			ans = temp;
			break;
		
	}
	printf("%d",ans);
}

File: gljive.py
import sys
num = sys.stdin.readlines()
ans = 0
i = 0
while i < 10:
	if ans < 100 :
		temp = ans
		ans += int(num[i])
		i += 1
	elif ans > 100 and (100 - temp) >= (ans - 100):
		print ans
		break
	elif ans > 100 and (100 - temp) < (ans - 100):
		print temp
		break
if ans <= 100:
	print ans